boards/shields/adafruit_2_8_tft_touch_v2/adafruit_2_8_tft_touch_v2.overlay

```diff
@@ -9,7 +9,7 @@
 &arduino_spi {
 	status = "okay";
 	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>,	/* D10 */
-		   <&arduino_header 12 GPIO_ACTIVE_LOW>;	/* D04 */
```

Comment:
I messed it up, for the [epaper shield](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/shields/waveshare_epaper/dts/waveshare_epaper_common.dtsi#L10) the comment is wrong but gpio pin is correct.